--- rxvt-unicode-9.17/doc/Makefile.in.orig	2013-03-03 09:52:13.923292379 +0100
+++ rxvt-unicode-9.17/doc/Makefile.in	2013-03-03 09:53:21.533289543 +0100
@@ -96,7 +96,7 @@
 	$(INSTALL_DATA) rxvtc.1.man $(DESTDIR)$(man1dir)/$(RXVTNAME)c.$(man1ext)
 	$(INSTALL_DATA) rxvtd.1.man $(DESTDIR)$(man1dir)/$(RXVTNAME)d.$(man1ext)
 	$(INSTALL_DATA) rxvt.7.man  $(DESTDIR)$(man7dir)/$(RXVTNAME).$(man7ext)
-	@TIC@ $(srcdir)/etc/rxvt-unicode.terminfo
+	@TIC@ -o $(srcdir) $(srcdir)/etc/rxvt-unicode.terminfo
 
 distdepend: alldoc
 
