diff -ur rxvt-unicode-9.09.orig/configure.ac rxvt-unicode-9.09/configure.ac
--- rxvt-unicode-9.09.orig/configure.ac	2010-11-25 14:08:18.762064681 +0100
+++ rxvt-unicode-9.09/configure.ac	2010-11-25 14:13:31.892064488 +0100
@@ -651,6 +651,7 @@
     AC_DEFINE(XFT, 1, Define to enable xft support)
   fi
 fi
+PKG_CHECK_MODULES([FC], [fontconfig])
 if test x$support_inheritpixmap = xyes; then
   support_xrender=yes
   AC_PATH_PROG(PKG_CONFIG, pkg-config, no)
diff -ur rxvt-unicode-9.09.orig/src/Makefile.in rxvt-unicode-9.09/src/Makefile.in
--- rxvt-unicode-9.09.orig/src/Makefile.in	2010-11-25 14:08:18.745397691 +0100
+++ rxvt-unicode-9.09/src/Makefile.in	2010-11-25 14:13:05.218731267 +0100
@@ -19,8 +19,8 @@
 LDFLAGS = @LDFLAGS@
 DEFS = @DEFS@
 LIBS = @LIBS@
-XINC = @X_CFLAGS@ @AFTERIMAGE_CFLAGS@ @PIXBUF_CFLAGS@
-XLIB = @X_LIBS@ @AFTERIMAGE_LIBS@ -lX11 @X_EXTRA_LIBS@ @PIXBUF_LIBS@
+XINC = @X_CFLAGS@ @AFTERIMAGE_CFLAGS@ @PIXBUF_CFLAGS@ @FC_CFLAGS@
+XLIB = @X_LIBS@ @AFTERIMAGE_LIBS@ -lX11 @X_EXTRA_LIBS@ @PIXBUF_LIBS@ @FC_LIBS@
 COMPILE = $(CXX) -I.. -I$(srcdir) -I. -I$(srcdir)/../libev $(DEFS) $(CPPFLAGS) $(CXXFLAGS) $(XINC)
 LINK = @LINKER@ $(LDFLAGS)
 EXEEXT = @EXEEXT@
