--- rxvt-unicode-5.9/autoconf/configure.in.orig	2005-12-26 15:43:00.804801000 +0100
+++ rxvt-unicode-5.9/autoconf/configure.in	2005-12-26 17:01:55.905956440 +0100
@@ -77,11 +77,11 @@
 AC_ISC_POSIX
 
 dnl# FreeBSD needs to link libxpg4
-AC_CHECK_LIB(xpg4, setlocale, [LIBS="$LIBS -lxpg4"])
+AC_SEARCH_LIBS(setlocale, xpg4)
 
 dnl solaris needs to link libnsl and socket
-AC_CHECK_LIB(nsl, gethostbyname, [LIBS="$LIBS -lnsl"])
-AC_CHECK_LIB(socket, socket, [LIBS="$LIBS -lsocket"])
+AC_SEARCH_LIBS(gethostbyname, nsl)
+AC_SEARCH_LIBS(socket, socket)
 
 dnl AC_ENABLE_SHARED(no)dnl#	libtool
 dnl AC_ENABLE_STATIC(yes)dnl#	libtool
