diff -Nur rxvt-unicode-9.18.orig/src/perl/tabbed rxvt-unicode-9.18/src/perl/tabbed
--- rxvt-unicode-9.18.orig/src/perl/tabbed	2013-02-17 00:41:38.000000000 +0000
+++ rxvt-unicode-9.18/src/perl/tabbed	2013-06-30 15:47:43.442909402 +0100
@@ -402,7 +402,7 @@
 # simply proxies all interesting calls back to the tabbed class.
 
 {
-   for my $hook qw(start destroy key_press property_notify) {
+   for my $hook (qw(start destroy key_press property_notify)) {
       eval qq{
          sub on_$hook {
             my \$parent = \$_[0]{term}{parent}
